operator/src/test/java/oracle/kubernetes/operator/MainTest.java

```diff
@@ -153,14 +168,6 @@ public void tearDown() throws Exception {
     mementos.forEach(Memento::revert);
   }
 
-  @SuppressWarnings("unchecked")
-  private Map<String, NamespaceStatus> getNamespaceStatusMap()
-          throws NoSuchFieldException, IllegalAccessException {
-    Field field = DomainNamespaces.class.getDeclaredField(NAMESPACE_STATUS_MAP);
-    field.setAccessible(true);
-    return (Map<String, NamespaceStatus>) field.get(null);
-  }
-
   private Map<String, NamespaceStatus> createNamespaceStatuses() {
```

Comment:
I think createNamespaceStatuses() can be removed.

Reply:
Yep, you're right - there's a few of these that can go away.